lib/removeUnusedVertices.js

```diff
@@ -353,14 +366,24 @@ function updateIndices(bufferViews, buffers) {
                 var indexStride = bufferView.extras._pipeline.indexStride;
                 var offsetArrayLength = offsetArray.length;
                 while (sourceIndex < viewEnd) {
-                    var index = (indexStride === 2) ? indexSource.readUInt16LE(sourceIndex) : indexSource.readUInt8LE(sourceIndex);
+                    var index = 0;
+                    switch(indexStride) {
+                        case 2:
+                            index = indexSource.readUInt16LE(sourceIndex);
+                            break;
+                        case 1:
+                            index = indexSource[sourceIndex];
+                            break;
+                    }

```

Comment:
We should probably have a helper function somewhere for reading/writing indexes like this, especially so it's easy to add uint32 support later. I noticed similar logic is in `combinePrimitives` as well. @lasalvavida have you done anything like this in your branch?